pkg/startup/startup.go

```diff
@@ -27,6 +27,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/juju/errors"
```

Comment:
Actually, I think this import is strange too - what is this library?

Reply:
Should be `k8s.io/apimachinery/pkg/api/errors` for `errors.IsNotFound` instead